CHANGELOG.md

```diff
@@ -4,32 +4,80 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Unreleased
+## 2.8.0 (2022-08-30)
+
+This release adds major new features since the 2.7.2 release.
+We deem it moderate priority for upgrading.
+
+This release includes these noteworthy features:
+
+* time_bucket now supports bucketing by month, year and timezone
+* Improve performance of bulk SELECT and COPY for distributed hypertables
+* 1 step CAgg policy management
+* Migrate Continuous Aggregates to the new format
```

Comment:
Should we add the name of the new function introduced to migrate: `cagg_migrate_to_finals_form` ?